OnlineShop/OnlineShoppingAngular/src/app/Services/ProductServices.service.ts

```diff
@@ -1,14 +1,35 @@
 import { HttpClient } from "@angular/common/http";
 import { Injectable } from "@angular/core";
 import { from, Observable } from "rxjs";
+import { BookParameter } from "../Classes/BookParameter";
+import { Product } from "../Classes/Product";
+import { ProductDto } from "../Classes/ProductDto";
 
 @Injectable()
 export class ProductService{
 
+     private BaseUrl = "https://localhost:44320/api/Book";
```

Comment:
This should be an environment variable, please make it configurable instead of hardcoding it.